src/components/selector-dropdown/__tests__/SelectorDropdown-test.js

```diff
@@ -550,56 +561,11 @@ describe('components/selector-dropdown/SelectorDropdown', () => {
             expect(wrapper.state('activeItemID')).toEqual(id);
         });
 
-        test('should call scrollItemIntoView', () => {
-            sandbox
-                .mock(instance)
-                .expects('scrollItemIntoView')
-                .once()
-                .withExactArgs(id);
+        test('should call scrollIntoView', () => {
+            const scrollIntoView = jest.spyOn(domUtils, 'scrollIntoView');
             instance.setActiveItemID(id);
-        });
-    });
 
-    describe('scrollItemIntoView()', () => {
```

Comment:
The component also does not have this function. It imports `scrollIntoView` which seems to cover this functionality.